js/battle.js

```diff
@@ -553,7 +556,7 @@ var Pokemon = (function () {
 			if (plaintext) {
 				name += ' (' + this.species + ')';
 			} else {
-				name += ' <small>(' + this.species + ')</small>';
+				name = '<span class="pokemon" title="' +  this.species + '">' + name + ' <small>(' + this.species + ')</small>' + '</span>';
```

Comment:
You may need to special-case tooltips, but log text should still work as I outlined it.